### DISCUSSION TITLE
New placeholder for current status of an island permission

### DISCUSSION THREAD

**Discussion by DuneSciFye** [BG-Software-LLC/SuperiorSkyblock2#1256] [Implemented Feature Requests] [answered]
a placeholder such as %superior_island_permission_status_<permission>%. the current %superior_island_permission_<permission>% simply returns if you are able to do x permission on current island. I'm looking for something that gives the permission status, like what value its set as, for example guest, mod, admin, visitor, or owner.

**Comment by OmerBenGera** [comment, marked as answer]
Added :)
